docs/src/pages/playground.tsx

```diff
@@ -0,0 +1,212 @@
+import React, { useEffect, useRef, useState } from 'react'
+import Layout from '@theme/Layout'
+import sdk from '@stackblitz/sdk'
+import styles from './playground.module.css'
+import { useColorMode } from '@docusaurus/theme-common'
+
+interface Project {
+	categoryTitle: string
+	subCategory: Array<CategoryItems>
+}
+
+interface CategoryItems {
+	name: string
+	templates: Array<Templates>
+}
+
+interface Templates {
+	title: string
+	desc: string
+	link: string
+	idx: number
+}
+
+function Playground() {
+	const [projectLink, setProjectLink] = useState('Wave-Play/robo.js/tree/main/templates/starter-app-js-react')
+	const [dropdown, setDropdown] = useState(false)
+	const [embedLoading, setEmbedLoading] = useState(true)
+	const { colorMode } = useColorMode()
+	const [fetchError, setFetchError] = useState<null | string>(null)
+	const [templates, setTemplates] = useState<Project[] | null>(null)
+	const [selectedTemplateIndex, setSelectedTemplateIndex] = useState(0)
+
+	const embedDiv = useRef<HTMLIFrameElement | null>(null)
+	const editor = useRef(null)
+
+	useEffect(() => {
+		const fetchTemplates = async () => {
+			const res = await fetch('http://localhost:3000/api/templates')
+
+			if (res.status !== 20) {
+				setFetchError(res.statusText)
+			}
+
+			if (res.ok && res.status === 200) {
+				const json = await res.json()
+				setTemplates(json)
+			}
+		}
+		fetchTemplates()
+	}, [])
+
+	useEffect(() => {
+		if (editor && editor.current) {
+			editor.current.setTheme(colorMode)
+		}
+	}, [colorMode])
+
+	const clickHandler = () => {
```

Comment:
The name of this function is too vague. I suggest using a more descriptive name such as `onClickDropdown`